### PR TITLE
added link to Quotefault in the Media section

### PR DESCRIPTION
**Pull request by dantayy**

### CODE REVIEW

File: data/links.json [hunk 1]
```diff
@@ -79,6 +79,11 @@
         "href": "https://segfault.csh.rit.edu/",
         "icon": "material:layers"
       }
```
Comment (mbillow) [inline data/links.json]: You are missing a comma here.

File: data/links.json [hunk 2]
```diff
@@ -79,6 +79,11 @@
...
+      {
+        "name": "Quotefault",
+        "href": "https://quotefault.csh.rit.edu/",
+        "icon": "material:quote"
```
Comment (mbillow) [inline data/links.json]: I am more partial to `foundation:comment-quotes`, I think it will look better with the rest of the icons.

Comment (dantayy) [inline data/links.json]: can do

File: data/links.json (hunk 1 above)
Comment (dantayy) [inline data/links.json]: that's how it was when i forked it, seems to be the case with all of the last items

Comment (mbillow) [inline data/links.json]: You added another hash to the list though, which means you have to add a comma between your addition and the previously last item.

Comment (dantayy) [inline data/links.json]: ahh i see the redline now, gotcha

**Pull request merged**